health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -263,16 +288,38 @@
     workbook.xlsx.writeBuffer();
 }
 
+function mergeErrors(errorData: any) {
+    const errorMap: any = {};
+
+    errorData.forEach((item: any) => {
+        const { rowNumber, sheetName, status, errorDetails, ...rest } = item;
+
+        // If the rowNumber already exists, merge the errorDetails
+        if (errorMap[rowNumber]) {
+            errorMap[rowNumber].errorDetails += "; " + errorDetails;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)